openstack/cronus/templates/cronus/config.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.cronus.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cronus
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    cloud.sap/skip-template: "true"
+  labels:
```

Comment:
The only label worth having is the app label. Not sure why the secret has these labels. My guess would be due to some helm migration, and needing them explictily there. But I would remove labels except for `app`